src/components/Goals.js

```diff
@@ -29,7 +46,10 @@ function Goals() {
   }
 
   return repository.issues ? (
-    <ListGoals goalsId={goalsId} goals={repository.issues} />
+    <Fragment>
```

Comment:
Instead of a `<Fragment>`, add the `<Card fitted />`
```suggestion
    <Card fitted>
```